source/features/remove-projects-tab.tsx

```diff
@@ -1,46 +1,75 @@
+/*
+The `Projects` tab is hidden from repositories, user profiles and organization profiles when there are no projects
+
+New projects can still be created via the [`Create new…` menu](https://user-images.githubusercontent.com/1402241/34909214-18b6fb2e-f8cf-11e7-8556-bed748596d3b.png).
+*/
+
 import React from 'dom-chef';
 import select from 'select-dom';
 import onetime from 'onetime';
 import features from '../libs/features';
-import {getRepoURL} from '../libs/utils';
 import {safeElementReady} from '../libs/dom-utils';
+import {isOwnUserProfile, isUserProfile} from '../libs/page-detect';
 
-const removeProjectsTab = () => {
-	// Only those who can create a project will see the 'Settings' tab
-	// so, do not remove the 'Projects' tab if the 'Settings' tab exists
-	if (select.exists('.js-repo-nav [data-selected-links^="repo_settings"]')) {
-		return false;
+const addNewProjectLink = onetime(() => {
+	if (isOwnUserProfile()) {
+		// The link already exists
+		return;
 	}
 
-	const projectsTab = select('.js-repo-nav [data-selected-links^="repo_projects"]');
-	if (projectsTab && projectsTab.querySelector('.Counter, .counter').textContent === '0') {
-		projectsTab.remove();
-		return true;
+	if (isUserProfile() && select.exists('[href*="contact/report-abuse?report="]')) {
```

Comment:
Should we move this logic into a `isOrgUserProfile()` method?

Reply:
Maybe. This would be `isOwnOrganizationProfile` (more correctly one you belong to)